src/Core/Models/Data/OrganizationUserOrganizationDetails.cs

```diff
@@ -1,8 +1,9 @@
 ﻿using System;
+using Bit.Core.Models.Interfaces;
 
 namespace Bit.Core.Models.Data
 {
-    public class OrganizationUserOrganizationDetails
+    public class OrganizationUserOrganizationDetails: IPermissions
```

Comment:
Need more space around your colon 😁 , ` : `

Reply:
Not applicable with IPermissions getting the boot